Issue #189 Standardize Minishift name

thatdocslady opened the pull request:
Fixed issue #189 

Reviewed and fixed a few places where Minishift had incorrect capitalization.

NOTE: For some reason there are files that I didn't touch in my work, I'm not sure if it's because of the master/cluster-up thing but I don't know how to withdraw them or if they need to be taken out? A bit confusing... When can we go back to just using master? cc @LalatenduMohanty @hferentschik

hferentschik commented:
@thatdocslady  You need to rebase your work on the _cluster-up_ branch.

    $ git checkout cluster-up
    # provided your upstream repo is called origin
    $ git pull origin cluster-up
    $ git checkout issue-189-naming
    $ git rebase cluster-up

    # if the rebase fails, you need to do an interactive rebase where you drop all of the commits except the last one
    $ git rebase -i cluster-up

hferentschik commented on CHANGELOG.md:
```diff
@@ -10,7 +10,7 @@
 * [BUG] Ensure node IP is routeable
 * [FEATURE] Reuse generated CA certificate
 * [FEATURE] Ensure xhyve driver uses same IP on restarts
-* [FEATURE] Add defaut insecure registry flag to include minishift service IP range
```
Personally I would not change the CHANGELOG in retrospective. This is part of the history.

LalatenduMohanty commented:
@thatdocslady @hferentschik I am not sure why this PR needs to be merged in to cluster-up branch. This PR has nothing to do with cluster up feature. This PR should go to master and `cluster-up` should rebase from master if required. 
 -1 for merging this to `cluster-up` branch

hferentschik commented:
>  I am not sure why this PR needs to be merged in to cluster-up branch. This PR has nothing to do with cluster up feature.

It needs to be on _cluster-up_, since it based on all the doc changes which occurred before. This change  does not even apply on _master_.

> -1 for merging this to cluster-up branch

This is the only option.

thatdocslady commented:
@hferentschik I rebased against upstream/cluster-up and the other commits are still showing up in this PR. Could it be the the commits weren't merged somehow before? Or am I doing something wrong? Also, changed back the CHANGELOG, I don't have strong opinions on this.

thatdocslady commented:
@hferentschik I think I know what the problem is, my fork (origin) doesn't have the cluster-up branch, and I fetched + pulled from upstream/cluster-up but then for some reason I can't push to origin. Maybe you can help me tomorrow, it's late and my brain is frazzled :(

hferentschik commented:
> Maybe you can help me tomorrow, 

Sure, let's go through this tomorrow together and apply some git foo.

hferentschik commented:
Merged, thanks.

thatdocslady commented:
@hferentschik I see this is closed with unmerged commits?
